tests/release/pipelines/push_to_external_registry.go.other

```diff
@@ -0,0 +1,196 @@
+package pipelines
```

Comment:
I guess this file should be removed also, right?

Reply:
and the magefile change